### ISSUE TITLE
polcoeff and pol_length not defined for absolute series

### ISSUE THREAD

**Issue by wbhart** [Nemocas/AbstractAlgebra.jl#783]
The documentation doesn't mention that these functions are currently only available for relative series.

Currently the absolute series generic code uses coeff and length instead of polcoeff and pol_length.

It would perhaps be better to define polcoeff and pol_length for absolute series too so the interface is more similar or to document the difference.

It might also make sense to rename polcoeff to pol_coeff.

**Comment by fieker** [comment]
On Thu, Mar 11, 2021 at 02:56:05AM -0800, wbhart wrote:
> The documentation doesn't mention that these functions are currently only available for relative series.
> 
> Currently the absolute series generic code uses coeff and length instead of polcoeff and pol_length.
> 
> It would perhaps be better to define polcoeff and pol_length for absolute series too so the interface is more similar or to document the difference.
> 
> It might also make sense to rename polcoeff to pol_coeff.
> 
setpolcoeff is also missing...(if we can have wishes - not that I needed
it)
in Hecke/examples/MPolyFact.jl

are a bunch of uses of this interface, mostly map_coeff

**Comment by wbhart** [comment]
Yeah there's just setcoeff! for both rel and abs series. It's probably too late for this (it'd only be used internally in the implementation itself).
